axes/conf.py

```diff
@@ -53,3 +53,11 @@ class MyAppConf(AppConf):
 
     # message to show when locked out and have cooloff disabled
     PERMALOCK_MESSAGE = 'Account locked: too many login attempts. Contact an admin to unlock your account.'
+
+    # set to the amount of proxies you are using if you have reverse proxies configured
+    IPWARE_PROXY_COUNT = None
+
+    # set to a trusted header value if your reverse proxy is configured to set it to the HTTP request
+    IPWARE_REQUEST_HEADER_ORDER = (
+        'REMOTE_ADDR',
+    )
```

Comment:
This leaves a project with two settings to configure:
* `IPWARE_REQUEST_HEADER_ORDER`
* `AXES_IPWARE_REQUEST_HEADER_ORDER`

I'd rather see secure defaults in django-ipware, because now axes is "forking" the settings to protect itself, whil leaving other packages to their fate. Before we know it, we have a `ALLAUTH_IPWARE_REQUEST_HEADER_ORDER` and `RATELIMIT_IPWARE_REQUEST_HEADER_ORDER`. If you'd still take this rate, at least use a configured `settings.IPWARE_REQUEST_HEADER_ORDER` as fallback in case it's configured. As this is such security point, this should use a really simple solution, and not a hairy set of settings to configure.

Reply:
I agree, but I've struggled enough to get the point across as it is and this is a workable solution.

Django Axes use to have it's own settings - albeit named differently - for these things when it had it's own implementation, so it isn't the worst thing.

This also does have the minor benefit that Axes can have it's own isolated configuration - someone may want to configure IPware to use a different header to look up a different IP for some other reason...